src/main/charts/bitbucket/values.yaml

```diff
@@ -752,6 +752,178 @@ bitbucket:
     #
     upstreamUrl:
 
+  # Mesh configuration
+  mesh:
+
+    # -- Enable Bitbucket Mesh. See: https://confluence.atlassian.com/bitbucketserver/bitbucket-mesh-1128304351.html
+    #
+    enabled: false
+
+    # -- Experimental! Automatically register Bitbucket mesh nodes with the Bitbucket server.
+    # `bitbucket.sysadminCredentials.secretName` needs to be defined to provide credentials
+    # to post-install node registration jobs that are created only for new Helm chart installations.
+    # It is recommended to manually register Mesh nodes in Butbucket UI.
+    #    
+    nodeAutoRegistration: false
+
+    # -- Experimental! Automatically create all new repositories on Bitbucket mesh nodes.
+    # `bitbucket.sysadminCredentials.secretName` needs to be defined to provide credentials
+    # to node post-install job. It is recommended to manually configure it in Bitbucket UI.
+    #
+    setByDefault: false
+
+    # -- The Bitbucket Mesh image to use
+    # https://hub.docker.com/r/atlassian/bitbucket-mesh
+    #
+    image:
+
+      # -- The Bitbucket Mesh image repository
+      # https://hub.docker.com/r/atlassian/bitbucket-mesh
+      #
+      repository: atlassian/bitbucket-mesh
+
+      # -- Image pull policy
+      #
+      pullPolicy: IfNotPresent
+
+      # -- The docker image tag to be used - defaults to the Chart appVersion
+      #
+      tag: "1.3.0"
```

Comment:
TODO: Once 1.5.0 is pushed to DockerHub, update the version. 1.3.0 is the latest available.